### PR TITLE
[sonic-gnmi] Support GNMI native write

### PR DESCRIPTION
**Pull request by ganglyu**
Signed-off-by: Gang Lv ganglv@microsoft.com

#### Why I did it
Provide GNMI native write interface for configuration.

#### How I did it
Add configuration parameters for GNMI native write.

#### How to verify it
Check build pipeline.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

#### Description for the changelog

Provide GNMI native write interface for configuration.

### CODE REVIEW

File: src/sonic-gnmi/.gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+build/
```
Comment (lguohan) [inline src/sonic-gnmi/.gitignore:1]: please create new repo sonic-gmni under sonic-net org and use submodule approach.

**Comment by zhangyanzhao** [comment]
@qiluo-msft  would you please review and approve this PR?
@ganglyu  please fix the build conflict.

**Review by liuh-80 — APPROVED** [review]
Because this PR depends on another submodule PR, suggest run validation again after submodule PR merged to master branch to make sure not conflict or build break.

**Review by zbud-msft — APPROVED** [review]

**Pull request merged**